src/App.css

```diff
@@ -0,0 +1,108 @@
+header {
```

Comment:
Please be sure that your design is almost a replica to the one provided as a guideline 🤓

Guidelines
![Captura](https://user-images.githubusercontent.com/15807343/187821671-d5422e1b-b9e5-42a9-af6f-6c9decedd3f9.PNG)

Kindly add the user icon and the highlighted area of the books so that your project meet the requirements 😉